TextRender.md

```diff
@@ -0,0 +1,162 @@
+# TextRender
+
+Generating structured text is best done with [`CAF::TextRender`][caf_textrender_docs].
+This document guides through the usage and testing of `CAF::TextRender`.
+
+Using `ncm-metaconfig`, which is the metacomponent build around
+`CAF::TextRender`, is described [here][metaconfig].
+
+TODO add correct/final url
+
+[caf_textrender_docs]: http://docs-test-caf.readthedocs.org/en/latest/CAF/CAF::TextRender
+[metaconfig]: https://github.com/quattor/configuration-modules-core/Metaconfig.md
+
+# CAF::TextRender
+
+Basic usage has 2 main modes:
+ * generate text : the `CAF::TextRender` instance has auto-stringification
+
+    <!-- language: lang-perl -->
```

Comment:
i'm keeping the xml tags for now
see https://github.com/quattor/release/issues/86
this is each to search and replace later